packages/components/src/select-control/README.md

```diff
@@ -114,7 +114,7 @@ Render a user interface to select multiple users from a list.
 		this.setState( { users } );
 	} }
 	options={ [
-		{ value: null, label: 'Select a User', disabled: true },
```

Comment:
`null` is not a valid value. Caught by TS in Storybook ☺️

Reply:
So good! That's exactly what TypeScript is great at.